framework/wazuh/core/cluster/master.py

```diff
@@ -737,10 +742,25 @@ async def sync_integrity(self, task_id: str, received_file: asyncio.Event):
                                       cls=c_common.WazuhJSONEncoder).encode()
                 await self.send_request(command=b'syn_m_c_r', data=task_id + b' ' + exc_info)
             finally:
+                try:
+                    # Try to remove task ID from the interrupted set.
+                    self.interrupted_tasks.remove(task_id)
+                    # Decrease max zip size if task was interrupted
+                    self.current_zip_limit = max(self.cluster_items['intervals']['communication']['min_zip_size'],
+                                                 sent_size * 0.9)
```

Comment:
Add 0.9, 0.8 and 1.1(As 1/0.9) to a variable in cluster.json, in the communications section.